### ISSUE TITLE
Paths Changed in VST SDK

### ISSUE THREAD

**Issue by Tony-Pappas** [ValdemarOrn/SharpSoundDevice#1]
The paths have changed in Steinberg's VST SDK. This has broken SharpSoundDevice - can't build it as per the SharpSoundDevice instructions. Even after modifying the paths (to the best of my abilities) the Build.bat still throws errors like:

fatal error C1083: Cannot open include file: 'pluginterfaces/vst2.x/aeffect.h': No such file or directory [SharpSoundDevice\SharpSoundDevice.VST\SharpSoundDevice.VST.vcxproj]

**Comment by ValdemarOrn** [comment]
I'm currently investigating issues I'm seeing when compiling a more recent version of the VSTSDK. I'm having problems loading the VST in any host, could be the issue you were seeing.

**Comment by ValdemarOrn** [comment]
This has now been resolved.
A new VstPluginMain method has been added in VstPluginBridge and vstpluginmain.cpp from the SDK is no longer used. This removes the DllMain entrypoint, which was causing issues with the CLR integration into the process. This may be the cause of the problems you were seeing in your [other issue](https://github.com/ValdemarOrn/SharpSoundDevice/issues/3). DllMain isn't strictly necessary in the VST standard, but is implemented by default on Windows platforms. After this change the only exported function is VSTPluginMain.

See: https://msdn.microsoft.com/en-us/library/ms173266.aspx

"Code within DllMain must not access the CLR. This means that DllMain should make no calls to managed functions, directly or indirectly; no managed code should be declared or implemented in DllMain; and no garbage collection or automatic library loading should take place within DllMain."

The readme.md has also been updated to show what the new expected layout should be.
I've also added v1.5.0.1 as a new build which is built using the latest SDK, and has DllMain removed.
